apps/minifront/src/components/swap/swap-form/simulate-swap-button.tsx

```diff
@@ -6,36 +6,36 @@ import {
   TooltipContent,
 } from '@penumbra-zone/ui/components/ui/tooltip';
 import { cn } from '@penumbra-zone/ui/lib/utils';
-import { AllSlices } from '../../../../state';
-import { useStoreShallow } from '../../../../utils/use-store-shallow';
+import { AllSlices } from '../../../state';
+import { useStoreShallow } from '../../../utils/use-store-shallow';
 
 const simulateSwapButtonSelector = (state: AllSlices) => ({
-  loading: state.swap.simulateOutLoading,
-  simulateSwap: state.swap.simulateSwap,
+  loading: state.swap.instantSwap.simulateSwapLoading,
+  simulateSwap: state.swap.instantSwap.simulateSwap,
+  disabled: state.swap.txInProgress || !state.swap.amount,
 });
 
 export const SimulateSwapButton = () => {
-  const { loading, simulateSwap } = useStoreShallow(simulateSwapButtonSelector);
+  const { loading, simulateSwap, disabled } = useStoreShallow(simulateSwapButtonSelector);
 
   return (
     <div className='grow'>
       <TooltipProvider>
         <Tooltip>
-          <TooltipTrigger className='w-full'>
-            <div
```

Comment:
Turns out that the `<div />` wasn't needed — we could just apply the classes directly to `<TooltipTrigger />`